crates/nassun/Cargo.toml

```diff
@@ -28,9 +28,12 @@ ssri = { workspace = true }
 thiserror = { workspace = true }
 tracing = { workspace = true }
 url = { workspace = true }
+mockall = "0.11.3"
 
 [dev-dependencies]
+mockito = "1.0.0"
 test-case = "3.0.0"
+tokio = { version = "1.26.0", features = ["macros"] }
```

Comment:
Two things:

1. Please add dependency versions directly to the workspace versions table in the root `Cargo.toml`, and consume them here with `{ workspace = true }`
2. You don't need tokio here. `[async_std::test]` should work well enough.

Reply:
1. moved to global Cargo.toml
2. Yes, but... I've had to change the `async-std` dependency to include the `tokio1`  feature. Otherwise `reqwest` will fail because it is `tokio` only. I don't exactly understand in detail, which changes this brings, so please have a more detailed look, if you can.

Reply:
that's fine. Having to add `tokio1` is to be expected :)